docs/components/CONTAINER_PRESENTER.md

```diff
@@ -112,11 +72,9 @@ passengers/
 
 ## Guidelines summary
```

Comment:
Should we move it at the beginning of the page ? People will first care if it fits their use case before the implementation details

Reply:
Erratum: seems more just guidelines or advices than a summary -- we can probably leave it here, but since we don't recommend it for all the use cases, we might need to make it clearer in the introduction